src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -11,6 +11,148 @@ namespace System.Diagnostics.Tests
     [Collection("System.Diagnostics.Debug")]
     public class DebugTests
     {
+        // [Fact] // uncomment when running this xunit method alone
```

Comment:
It would be nice to add links to github issues here and just disable the test on .NET Core against those bugs.

Reply:
Added links to issue. Thanks